apps/opik-backend/src/main/java/com/comet/opik/domain/sorting/SortingQueryBuilder.java

```diff
@@ -0,0 +1,22 @@
+package com.comet.opik.domain.sorting;
+
+import com.comet.opik.api.sorting.SortingField;
+import lombok.NonNull;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SortingQueryBuilder {
+    public String toOrderBySql(@NonNull List<SortingField> sorting) {
+        if (sorting.isEmpty()) {
+            return null;
+        }
+
+        return sorting.stream()
+                .map(sortingField -> {
+                    String sortOrder = sortingField.desc() ? "DESC" : "ASC";
```

Comment:
I'm not sure if you saw Andres's comment about using more like an enum instead of the boolean.

Reply:
Fixed